### PR TITLE
Update vesting documentation

### PR DESCRIPTION
**Pull request by sergeyshemyakov**

### CODE REVIEW

**Review by Phanco — APPROVED** [review]

File: documentation/vesting.md [hunk 1]
```diff
@@ -10,20 +10,21 @@ The following table gives an overview of the vesting schedules, each implemented
...
+| Team II    | L1            | 1,500,000 |[0x2294A7f24187B84995A2A28112f82f07BE1BceAD](https://eth.blockscout.com/address/0x2294A7f24187B84995A2A28112f82f07BE1BceAD)| [0x586C7735d78f421495FE7e4E32B4e13a90661395](https://eth.blockscout.com/address/0x586C7735d78f421495FE7e4E32B4e13a90661395) | 1735689600 | 1461 | All tokens are linearly released over 4 years between 1.1.2025 and 31.12.2028. |
+| Investors | L1            | 30,000,000 |[0x58a61b1807a7bDA541855DaAEAEe89b1DDA48568](https://eth.blockscout.com/address/0x58a61b1807a7bDA541855DaAEAEe89b1DDA48568)| [0x4B00A4659454013388b39DF9b23F5DbE65Bbc06E](https://eth.blockscout.com/address/0x4B00A4659454013388b39DF9b23F5DbE65Bbc06E) | 1716163200 | 730 | 10,000,000 LSK liquid at migration, the remaining 20,000,000 LSK vested linearly over 24 months, starting 20.05.2024. |
```
Comment (janhack) [inline documentation/vesting.md:18]: Only 20 million of the investor tokens are vested.

File: documentation/vesting.md [hunk 2]
```diff
@@ -10,20 +10,21 @@ The following table gives an overview of the vesting schedules, each implemented
...
+| Ecosystem Fund | L1            | ~7,860,000 |[0x114cB34b1A0fBBB686E31Bf5542d64A98c42eE23](https://eth.blockscout.com/address/0x114cB34b1A0fBBB686E31Bf5542d64A98c42eE23)| [0xC30B50cdCccEb70b8D87fDda4F08258e8A02539E](https://eth.blockscout.com/address/0xC30B50cdCccEb70b8D87fDda4F08258e8A02539E) | 1735689600 | 0 | 5,000,000 LSK are liquid at migration, the remaining ~2,860,000 LSK are released on 1.1.2025. |
```
Comment (janhack) [inline documentation/vesting.md:19]: We know the exact numbers now, `2,711,644` are vested and the total is `7,711,644`.

**Review by janhack — CHANGES_REQUESTED** [review]
Looks good, just one small point.

File: documentation/vesting.md (hunk 2 above)
Comment (sergeyshemyakov) [reply documentation/vesting.md]: Fixed

**Review by janhack — APPROVED** [review]

**Comment by matjazv** [comment]
This PR was duplicated. Valid PR is here: https://github.com/LiskHQ/lisk-contracts/pull/189

**Pull request closed without merging**